iocage/cli/set.py

```diff
@@ -66,15 +71,15 @@ def cli(ctx, props, jail):
 
     updated_jail_count = 0
 
-    for jail in ioc_jails:
+    for ioc_jail in ioc_jails:  # type: iocage.lib.Jail.JailGenerator
```

Comment:
how the heck does that need a hint?? that feels like a bug in mypy

Reply:
You can try to remove. Being explicit in other places should have fixed it anyway!

Reply:
you're right
works without it, too.